modules/express/src/clientRoutes.ts

```diff
@@ -461,11 +461,76 @@ export async function handleV2Sign(req: express.Request) {
   }
 }
 
+export async function handleV2OFCSignPayloadInExtSigningMode(
+  req: express.Request
+): Promise<{ payload: string; signature: string }> {
+  const walletId = req.body.walletId;
+  const payload = req.body.payload;
```

Comment:
Nit: we can define a codec for this request and use io-ts.

Reply:
Thanks for the suggestion! This feature is currently blocking testing with a partner and so I want to keep the scope for this ticket as low as possible and get it over the finish line ASAP. Will keep this in mind for a future iteration on this feature.